lib/grpc/client/adapters/mint.ex

```diff
@@ -119,12 +125,22 @@ defmodule GRPC.Client.Adapters.Mint do
       |> Keyword.get(:transport_opts, [])
       |> Keyword.merge(ssl)
 
-    [transport_opts: Keyword.merge(@default_transport_opts, transport_opts)]
+    client_settings = Keyword.get(opts, :client_settings, @default_client_settings)
+
+    [
+      transport_opts: Keyword.merge(@default_transport_opts, transport_opts),
+      client_settings: client_settings
```

Comment:
It might be useful to update the module documentation to mention the new client_settings option and its defaults. This will help users to understand the new configurations they can use and modify if needed.

Reply:
Ohhh yeah -- M'bad. I'll work on this later today

Reply:
Check [2f4623a](https://github.com/elixir-grpc/grpc/pull/378/commits/2f4623a86e5b8ece04ec9e032a63c1f3e1347f05) and let me know if this is helpful enough. I tried to keep short, but still informative. I can expand if needed